seagap/configs/solver.py

```diff
@@ -37,6 +39,42 @@ class ArrayCenter(BaseModel):
     lon: float = Field(..., description="Longitude")
 
 
+class InputData(BaseModel):
```

Comment:
Could this and `OutputPath` be the same type (e.g. `FileConfig`)? They look very similar. The differences appear to be: a) The use of fsspec for output (should this be used for input too?) and validation that the input exists (could this be a parameter, e.g. or we could have a derived `RequiredFileConfig` that just adds that validation.

If we do need more than one type for I/O file config then can they be in the same file?

Reply:
They're very similar, but behave a bit different. They both does use `fsspec`. The input will check for if the file path exists, but the output will check if we're able to write to that path. Initially, I was thinking like I can combine this two, however since the descriptions for the `path` attribute is different, as well as the behavior during the instantiation, I separated them to two different models.

> If we do need more than one type for I/O file config then can they be in the same file?

Do you mean create a module for these two to live?

Reply:
Right, just in terms of code organization I'd expect these two to be close to each other (ie. same file or module)